### PR TITLE
chore[test]: modify `blobhash` test

### PR DESCRIPTION
**Pull request by tserg**
Modify blobhash test

### CODE REVIEW

File: tests/functional/builtins/codegen/test_blobhash.py [hunk 1]
```diff
@@ -45,46 +47,63 @@ def test_blobhash_success(good_code):
...
+    # print(blob_transaction.blob_data.versioned_hashes)
+    expected_versioned_hash = "0x0168dea5bd14ec82691edc861dcee360342a921c1664b02745465f6c42239f06"
+
+    def _send_tx_with_blobs(num_blobs, input_idx):
+        text = b"Long live the BLOBs!"
+        # BLOBs contain 4096 32-byte field elements.
+        # (32 * 4096) / 2 ** 10 = 128.0 -> Each BLOB can store up to 128kb.
+        blob_data = text.rjust(32 * 4096)
+
+        sig = keccak("set_blobhash(uint256)".encode()).hex()[:8]
```
Comment (pcaversaccio) [inline tests/functional/builtins/codegen/test_blobhash.py:74]: can't we use simply `function_signature_to_4byte_selector` here?

```py
from eth_utils import function_signature_to_4byte_selector

sig = function_signature_to_4byte_selector('set_blobhash(uint256)')
```

**Review by pcaversaccio — APPROVED** [review]

File: tests/functional/builtins/codegen/test_blobhash.py (hunk 1 above)
Comment (tserg) [reply tests/functional/builtins/codegen/test_blobhash.py:74]: that would work too, but the current test suite does not rely on this and seems to standardize to manually constructing the function sig using the keccak fixture. 

Examples.
- https://github.com/vyperlang/vyper/blob/097aecfe29e6c218d165d223f0272265cf6f5403/tests/functional/codegen/features/decorators/test_payable.py#L187
- https://github.com/vyperlang/vyper/blob/097aecfe29e6c218d165d223f0272265cf6f5403/tests/functional/builtins/codegen/test_raw_call.py#L262

I think we could discuss whether we want to switch to `eth_utils.function_signature_to_4byte_selector` across the test suite separately.

Comment (pcaversaccio) [reply tests/functional/builtins/codegen/test_blobhash.py:74]: gotcha - I just remembered this utility and thought using it, but nothing blocking. I pinged @charles-cooper to quickly review this PR and will merge after. Thanks for it!

Comment (tserg) [reply tests/functional/builtins/codegen/test_blobhash.py:74]: np, thanks ser!

**Pull request merged**